README.md

```diff
@@ -243,10 +242,17 @@ OpenapiConfig:
     ServingDirectory: api/swagger-ui-dist
     ServingPath: /api-docs/
 ```
+Before we can start turing, we need to get the credentials of the K3d cluster created in the docker-compose step. 
+```
+sh ../infra/docker-compose/dev/extract_creds.sh
+```
+This script will create 2 new files containing the credentials required to configure turing with:
+* config-dev-w-creds.yaml
```

Comment:
Should these files be added to `.gitignore`?

Also, can we briefly describe whats the difference between these 2 newly generated files?

Reply:
Added.